src/adapters/script_runner.cpp

```diff
@@ -74,7 +78,7 @@ bool script_runner::is_stopping() const {
  *
  * Returns true if the output changed.
  */
-bool script_runner::set_output(const string&& new_output) {
+bool script_runner::set_output(string&& new_output) {
```

Comment:
I don't even know why this code ever compiled before. Looking at it again, `std::move(new_output)` should have produced a warning since it only takes a non-const rvalue reference.

Reply:
I believe copy constructor was used in this case, since it's a better match then non-const rvalue version, thus no warning (a suitable candidate was found). Clang-tidy warns about it though, that's how i discovered this in the first place.

Reply:
That makes sense. I really need to add clang-tidy to the CI to catch this sort of thing.